pyrefly.toml

```diff
@@ -1,6 +1,6 @@
 # A Pyrefly configuration for PyTorch
 # Based on https://github.com/pytorch/pytorch/blob/main/mypy.ini
-python-version = "3.12"
+python-version = "3.10"
```

Comment:
This also caused some cascading changes; since 3.10 is the min version supported though I think this is a necessary change for us and that we should keep this tied to our min version